executor/distsql.go

```diff
@@ -245,7 +245,9 @@ func (e *IndexReaderExecutor) Close() error {
 		copStats := e.ctx.GetSessionVars().StmtCtx.RuntimeStatsColl.GetRootStats(e.plans[0].ExplainID().String())
 		copStats.SetRowNum(e.feedback.Actual())
 	}
-	e.ctx.StoreQueryFeedback(e.feedback)
+	if e.ctx.GetSessionVars().StmtCtx.InSelectStmt {
```

Comment:
Why do we need to double-check here?
IMHO, we only need to decide whether to fetch feedback during planning?

Reply:
Same reason, `explain analyze` may generate feedbacks we don't want to dump.